kotlin/show-location-history/README.md

```diff
@@ -0,0 +1,42 @@
+# Show location history
+
+Display your location history on the map.
+
+![Image of show location history](show-location-history.jpg)
+
+## Use case
+
+You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. An unmapped trail, for example, could be added to the map using this technique.
+
+## How to use the sample
+
+Tap the floating action button to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Tap the button again to stop updating the location history. This sample uses a simulated data source to allow the sample to be useful on desktop/non-mobile devices. To track a user's real position, use the `DefaultLocationDataSource` instead.
```

Comment:
Probably don't need the comment about desktop devices

Reply:
good point!